examples/howto/plot_connectivity.py

```diff
@@ -48,8 +48,7 @@
 print(net_erp.connectivity[conn_idx])
 plot_connectivity_matrix(net_erp, conn_idx)
 
-gid_idx = 11
-src_gid = net_erp.connectivity[conn_idx]['src_gids'][gid_idx]
+src_gid = net_erp.connectivity[conn_idx]['src_gids'].copy().pop()
```

Comment:
I just realized that the docs weren't getting built properly with the recent changes. Should I update the API in `whats_new.rst` as well?